packages/console/src/containers/AppBoundary/index.tsx

```diff
@@ -12,12 +12,11 @@ function AppBoundary({ children }: Props) {
   const {
     data: { language },
   } = useUserPreferences();
+  const { i18n } = useTranslation();
 
   useEffect(() => {
-    (async () => {
-      void initI18n(language);
-    })();
-  }, [language]);
+    void i18n.changeLanguage(language);
```

Comment:
Does it know what language to fallback if language is not loaded yet?

Reply:
type of `language` is a non-nullable string literal. the fallback logic is in `useUserPreferences()`